src/views/Affiliates/LastUpdated.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useStringGetter } from '@/hooks/useStringGetter';
+
+// Styled component for the container
+const Container = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  font-family: 'Inter', sans-serif;
+`;
+
+interface IProps {
+  lastUpdatedDate: Date;
+}
+
+const LastUpdated = ({ lastUpdatedDate }: IProps) => {
+  const stringGetter = useStringGetter();
+
+  // State to hold the current time
+  const [currentTime, setCurrentTime] = useState<Date>(new Date());
+
+  // Update the current time every minute
+  useEffect(() => {
+    const timer = setInterval(() => {
+      setCurrentTime(new Date());
+    }, 60000); // Update every 60 seconds
+
+    return () => clearInterval(timer); // Cleanup the timer
+  }, []);
+
+  // Helper function to calculate "X mins ago"
```

Comment:
for these time formating I wonder if we can leverage:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat

e.g
```
function timeAgo(timestamp: Date): string {
    const diffInSeconds = Math.floor((Date.now() - timestamp.getTime()) / 1000);

    const rtf = new Intl.RelativeTimeFormat('en', { numeric: 'auto' });

    if (diffInSeconds < 60) {
        return 'updated just now';
    } else if (diffInSeconds < 3600) {
        const minutes = Math.floor(diffInSeconds / 60);
        return `updated ${rtf.format(-minutes, 'minute')}`;
    } else if (diffInSeconds < 86400) {
        const hours = Math.floor(diffInSeconds / 3600);
        return `updated ${rtf.format(-hours, 'hour')}`;
    } else {
        const days = Math.floor(diffInSeconds / 86400);
        return `updated ${rtf.format(-days, 'day')}`;
    }
}
```